admin_interface/templates/admin/base_site.html

```diff
@@ -146,11 +146,13 @@
 {% if theme.list_filter_highlight %} list-filter-highlight {% endif %}
 {% if theme.list_filter_sticky %} list-filter-sticky {% endif %}
 
+{% if adminform %}
```

Comment:
Maybe it's better to do `{% if adminform and inline_admin_formsets %}`?

Reply:
That second variable is defined in the _changeform_view method, not in all admin views!

Reply:
I'm not understanding, it the first variable is defined in all admin views, why should check it?

Reply:
No, adminform is not defined in the login and logged out templates.

`if adminform` is a way to check if the variable is defined, not only if the object referenced is not falsey.

Reply:
Oh, I see your point! So yeah same thing can work for both.

Reply:
```suggestion
{% if adminform and inline_admin_formsets %}
```